tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{35,36}-{sqlite,postgres}
+envlist = py{36,37}-{sqlite,postgres}
```

Comment:
You can leave it as `py{35,36,37}...` and then just set:
`skip_missing_interpreters = True`
To skip that which you don't have installed.